### PR TITLE
adds a listing of common editor-directives and improves inspire settings

### PR DESCRIPTION
**Pull request by pvgenuchten**
@josegar74 please verify if all of these directives are available in master, else would be good to PR them. I'm not aware if these directives are iso19139 only, in that case maybe we should label them as such.

Related to INSPIRE, I added (almost) all thesauri which are mentioned in TG, not sure if they are currently used in config-editor to populate anchor fields, i can also imagine a development where we auto-complete the xlink:href from js-code or in update-fixed info (in that scenario, adding them as thesauri may not be required). Another consideration is that we could add a setting to thesauri to hide them in keyword-directives (in case it is added to support specific fields, eg protocol, crs, applicationschema, quality-report-type)

### CODE REVIEW

File: source/administrator-guide/configuring-the-catalog/inspire-configuration.rst [hunk 1]
```diff
@@ -30,87 +30,95 @@ When enabled, the INSPIRE support activate the following:
...
+* Metadata codelist register > `INSPIRE priority data set <https://inspire.ec.europa.eu/metadata-codelist/PriorityDataset>`_
+* Metadata codelist register > `Spatial Data Service Category <https://inspire.ec.europa.eu/metadata-codelist/SpatialDataServiceCategory>`_
+* Metadata codelist register > `Conditions applying to Access and Use <https://inspire.ec.europa.eu/metadata-codelist/ConditionsApplyingToAccessAndUse>`_
+* Metadata codelist register > `Limitations on Public Access <https://inspire.ec.europa.eu/metadata-codelist/LimitationsOnPublicAccess>`_
+* Metadata codelist register > `OnLine Description Code <https://inspire.ec.europa.eu/metadata-codelist/OnLineDescriptionCode>`_
+* Metadata codelist register > `Quality of Service Criteria <https://inspire.ec.europa.eu/metadata-codelist/QualityOfServiceCriteria>`_
+ 
+
+From ``admin console`` > ``classification systems`` > ``Thesaurus``, administrators can manage thesauri. One of the options is to load a thesaurus straight from the registry. 
+It is important to verify the thesaurus date, because in some case INSPIRE requires a specific date of the thesaurus. After import you can download the thesaurus, change it
```
Comment (fxprunayre) [inline source/administrator-guide/configuring-the-catalog/inspire-configuration.rst]: If there is a bug, we should fix it no ? What thesaurus loaded from Registry needs a date to be updated ? We have this in place here https://github.com/geonetwork/core-geonetwork/blob/master/web/src/main/webapp/xslt/services/thesaurus/registry-to-skos.xsl#L77-L84 so it is only  a matter of configuration. User reading that does not really know what to do ? What is the correct date ?

File: source/customizing-application/editor-ui/creating-custom-editor.rst [hunk 2]
```diff
@@ -889,3 +889,64 @@ Child elements:
 
 - **template**, Optional element (see :ref:`creating-custom-editor-template`)
 
+
+.. _common_advanced_editor_fields:
```
Comment (fxprunayre) [inline source/customizing-application/editor-ui/creating-custom-editor.rst]: Metadata editor doc is based on https://github.com/geonetwork/core-geonetwork/blob/master/schemas/config-editor.xsd so you need to add it here. Probably around https://github.com/geonetwork/core-geonetwork/blob/master/schemas/config-editor.xsd#L1744

I'll check if instruction are missing to build the rst file from the XSD.

File: source/administrator-guide/configuring-the-catalog/inspire-configuration.rst (hunk 1 above)
Comment (pvgenuchten) [reply source/administrator-guide/configuring-the-catalog/inspire-configuration.rst]: I understood that the problem is on the side of the eu registry, which is not able to store snapshots of a thesaurus by date. and will always return the current date.

Dates should be used as mentioned in TG

Yes, we could introduce a list in geonetwork to introduce the relevant date for each of the inspire thesauri, but i'd prefer this is solved at the registry (or in the guidelines, not to require a date)

Let me remove the section for now

File: source/customizing-application/editor-ui/creating-custom-editor.rst (hunk 2 above)
Comment (pvgenuchten) [reply source/customizing-application/editor-ui/creating-custom-editor.rst]: Do I understand correctly that 'creating-custom-editor.rst' is auto generated from 'config-editor.xsd'? So you suggest I move these changes to the xsd document?

**Comment by pvgenuchten** [comment]
superseded by #105

File: source/customizing-application/editor-ui/creating-custom-editor.rst (hunk 2 above)
Comment (fxprunayre) [reply source/customizing-application/editor-ui/creating-custom-editor.rst]: Yes using https://github.com/geonetwork/core-geonetwork/blob/master/docs/schema-doc/pom.xml which was probably only used once. Will try to resurrect it.

**Pull request closed without merging**